_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_hw_par.cpp

```diff
@@ -2032,7 +2032,12 @@ mfxStatus CheckVideoParam(MfxVideoParam& par, ENCODE_CAPS_HEVC const & caps, boo
     changed += CheckMax(par.mfx.NumRefFrame, maxDPB - 1);
 
     if (par.mfx.NumRefFrame)
-        maxDPB = par.mfx.NumRefFrame + 1;
+    {
+        // If NumActiveRef parameters are set already, DPB size should not be less than NumActiveRef+1
+        maxDPB = Max<mfxU16>(
```

Comment:
please, use std min/max primitives to not introduce conflicts for #925.

Here you can use ```std::max``` for initializer list:
```
maxDPB = std::max<mfxU16>({par.mfx.NumRefFrame + 1, par.m_ext.DDI.NumActiveRefP+1,
                               par.m_ext.DDI.NumActiveRefBL0+1, par.m_ext.DDI.NumActiveRefBL1+1});
```

Reply:
Max<> template is used all over the code in this source file, are you sure that's it's good idea to mix them with  std::max ? Please check. If it's still required, I'll change it.

Reply:
Done